domain/schema/controller.go

```diff
@@ -532,3 +533,46 @@ INSERT INTO autocert_cache_encoding VALUES
     (0, 'x509');    -- Only x509 certs encoding supported today.
 `)
 }
+
+func userSchema() schema.Patch {
+	return schema.MakePatch(`
+CREATE TABLE user (
+    uuid            TEXT PRIMARY KEY,
+    name            TEXT NOT NULL,
+    display_name    TEXT,
+    removed         BOOLEAN NOT NULL,
+    created_by_uuid TEXT,
+    created_at      TIMESTAMP NOT NULL,
+    CONSTRAINT      fk_user_created_by_user
+        FOREIGN KEY (created_by_uuid)
+    REFERENCES      user(uuid)
+);
+
+CREATE UNIQUE INDEX idx_singleton_active_user ON user (name) WHERE deleted IS FALSE;
+
+CREATE TABLE user_last_login (
+    user_uuid       TEXT PRIMARY KEY,
+    last_login      TIMESTAMP NOT NULL,
+    CONSTRAINT      fk_user_last_login_user
+        FOREIGN KEY (user_uuid)
+    REFERENCES      user(uuid)
+);
+
+CREATE TABLE user_password (
+    user_uuid       TEXT PRIMARY KEY,
+    password_hash   TEXT NOT NULL,
+    password_salt   TEXT NOT NULL,
+    disabled        BOOLEAN NOT NULL,
```

Comment:
Shouldn't this be on the `user` table?

Reply:
moved to the user table.

Reply:
I don't think disable should be on the user table. You are never disabling a user as that can only be done by removing the presence of a user.

What we are disabling is the users ability to authenticate with the api server. I think by that we should be keeping disable with the authentication information and not with the user.